packages/workers-shared/router-worker/src/index.ts

```diff
@@ -75,6 +77,69 @@ export default {
 
 			const maybeSecondRequest = request.clone();
 
+			if (config.static_routing) {
```

Comment:
Do we want to time/trace static routing evaluation?

Reply:
i don't think that's necessary, since it should only be on the order of milliseconds. at the very least we'd be able to infer static routing cost between the start of the request context and any dispatch span. but if you think it'd be an asset i'm happy to add it

Reply:
Nope, fine to not bother for now! We can always add in the future if we need to dig deeper. Just wanted to flag for consideration.